### PR TITLE
fix #137

### LINKED ISSUE #137: Shorts can now be 3 minutes long
The filter needs to be updated.

### PR DESCRIPTION
**Pull request by Fabian42**
Untested, just edited in GitHub with limited JavaScript knowledge, but I see no reason why this should fail… (:

### CODE REVIEW

File: sheetScript.gs [hunk 1]
```diff
@@ -583,24 +583,21 @@ function removeShortsFilter(videoId) {
     id: videoId,
   });
   if (response.items && response.items.length && response.items[0].contentDetails.duration) {
-    return !isLessThanAMinute(response.items[0].contentDetails.duration)
+    return !isLessThanThreeMinutes(response.items[0].contentDetails.duration)
   }
   return false;
 }
 
 // Checks if an ISO 8601 duration is less or equal than a minute.
```
Comment (victorjzsun) [inline sheetScript.gs:591]: Can you update this comment too?

**Review by victorjzsun — CHANGES_REQUESTED** [review]

File: sheetScript.gs (hunk 1 above)
Comment (Fabian42) [reply sheetScript.gs:591]: Is GitHub not showing the right thing? Line 586 is definitely edited and there is no comment there.

Comment (victorjzsun) [reply sheetScript.gs:591]: Sorry, I meant line 591, the last line of the snippet. GitHub UI could've made it more clear

Comment (Fabian42) [reply sheetScript.gs]: I could have sworn that I already did that. I did a Ctrl+F for "minute" and updated it all. Anyway, now it is done.

Comment (victorjzsun) [reply sheetScript.gs]: Thanks!

**Review by victorjzsun — APPROVED** [review]

**Comment by victorjzsun** [comment]
Thanks for the help!

**Pull request merged**